src/Commands/Tasks.h

```diff
@@ -30,6 +30,15 @@ String Command_Task_ValueSet(struct EventStruct *event, const char* Line)
 	return return_command_success();
 }
 
+String Command_Task_ValueToggle(struct EventStruct *event, const char* Line)
+{
+	const float result = UserVar[(VARS_PER_TASK * (event->Par1 - 1)) + event->Par2 - 1];
+	if (result == 0 || result == 1) {
```

Comment:
Floats cannot be compared like this.
Either cast to int or compare should include a range of about the resolution of a float

Reply:
you are right. sorry.